Assignment8.txt

```diff
@@ -0,0 +1,9 @@
+Ashank Kumar
+
+1. printf writes text to the ouput stream(monitor) while fprintf writes text to the output stream the programmer tell it to(a file).
+
+2. A file in "w" mode will overwrite anything in the file, while "a" will add to whatever is in the file. "rw" will first read to the file then write to it, so what ever is in the file wont be replaced. While "wr" fill overwrite whatever is in the file.
```

Comment:
right - the order of when the file is cleared is what differs between rw and wr.